.gitignore

```diff
@@ -65,3 +65,4 @@ test.sh
 .vscode/*
 /suricata-update/*
 !/suricata-update/Makefile.am
+/suricata-config
```

Comment:
should this now be changed to `/libsuricata-config`?

Reply:
Yes